### DISCUSSION TITLE
Detecting and Recording Tractor Trailer Trucks/Semi Trucks

### DISCUSSION THREAD

**Discussion by DC651F82053AD314** [blakeblackshear/frigate#8507] [General]
Hi!

I'm loving Frigate, it really beats using something like MotionEye, I get way less false positives in my testing with Frigate, using the USB Coral device.

I have a question - I'm currently getting ready to deploy a Frigate setup for a project where we're only interested in having our gate cameras detect when tractor trailer trucks are going through a gate.  Only tractor trailer trucks go through this gate.

In the Frigate yml settings file, I have enabled only 'car' and 'bus' (just in case it thinks these large semi trucks are buses), at [this link](https://docs.frigate.video/configuration/objects/) I saw it mentioned that "`car` is listed twice because `truck` has been renamed to `car` by default. These object types are frequently confused."

So, I just wanted to verify that I wouldn't need to put 'truck' in the object detection list, correct?  Would 'car' and 'bus' be enough to only capture tractor trailer trucks?  This isn't saying that one should list 'car' twice if they want the coral to pick up trucks too, is it?  Because I see 'car' in the object list twice, it's a bit confusing.

I have it pointed at my street for testing, but not too many semi/tractor trailor trucks head down my street, so I'm trying to get an idea ahead of time.

Thanks in advance for any answers or help anyone who knows these answers can provide!

**Comment by NickM-27** [comment]
Yes, all you need to do is put car once in your config as a tracked object

**Reply by DC651F82053AD314** [reply]
Thanks!  In your opinion, should I take 'bus' out as a tracked object?  My initial thought was that having 'bus' with 'car' might allow it to pick up the semi's when/if car doesn't, but I was also thinking it may cause double up recordings.  Then again, I'm sure there haven't been a whole lot of others with this use case, so I can always experiment once I deploy my new remote system.

**Reply by NickM-27** [reply]
In my experience with coco I never really saw bus get detected.

**Comment by gunnaraas** [comment]
I have [a similar use case](https://github.com/blakeblackshear/frigate/issues/11013) that I'm seeking suggestions for about best approach. 

I'd like to be able to distinguish between `car` object and `truck` objects. Unlike the gate you've mentioned, I'll be looking at a busy street. Cars are normal and expected, I'm instead looking to track freight vehicles that travel past this camera. Does Frigate / Frigate+ offer any solution to this?

**Reply by NickM-27** [reply]
There is no truck object in frigate+ though you could use car object along with a min size filter.
